src/pages/starter-kit/checkout/configure.md

```diff
@@ -82,6 +82,40 @@ To configure eventing, follow these steps:
     - Update the `events_of_interest` field with the events that your app requires.
     - Update the `runtime_action` field with the action that will consume the events.
 
+1. To configure multiple Commerce event providers, update your configuration as follows:
```

Comment:
This section can be shortened and moved under items #3 and #5, as they already describe all the required steps for preparing events.config.yaml and app.config.yaml, simplify saying that 'update <your_application_name> in each file'. And move the detailed explanation to [the file description section](https://github.com/AdobeDocs/commerce-extensibility/blob/d10b713ac00707a03e619c8cdc7f2e3a26dacf4a/src/pages/starter-kit/checkout/configure.md#appconfigyaml) if needed.

Also, if you have any link from Commerce Eventing to attach here, that would be great.